subsys/bluetooth/host/gatt.c

```diff
@@ -425,7 +425,8 @@ enum {
 #define CF_BIT_NOTIFY_MULTI	2
 #define CF_BIT_LAST		CF_BIT_NOTIFY_MULTI
 
-#define CF_BYTE_LAST		(CF_BIT_LAST % 8)
+#define CF_NUM_BITS             (CF_BIT_LAST + 1)
```

Comment:
I don't see this used anywhere. Should it be?

Reply:
I saw that too, updated with where I wanted to use it :)